docs/howto/operate/move-hub.md

```diff
@@ -200,6 +221,27 @@ This step preserves user information, since they might be added via the admin UI
    kubectl --namespace NEW_NAMESPACE cp -c hub ./jupyterhub.sqlite NEW_HUB_POD_NAME:/srv/jupyterhub/jupyterhub.sqlite
    ```
 
-## Transfer DNS
+## 6. Delete all user pods from the old cluster
+
+This will kick out all users from the old hub and close any running user servers and kernels.
+
+```bash
+kubectl get pods -n <old_prod_namespace> --no-headers=true | awk '/jupyter/{print $1}' | xargs kubectl delete -n <old_prod_namespace> pod
+```
+
+## 7. Do one final copying of the home directories
+This will catch all the user changes since the last rsync. However long this takes is the total amount of downtime we'll have.
+
+## 8. Transfer DNS
+
+Retrieve the external IP address for the `ingress-nginx` load balancer.
+
+```bash
+kubectl --namespace support get svc support-ingress-nginx-controller
+```
+
+Edit the existing DNS entry in NameCheap that matches the old hub domain and type in this external IP address.
+
+This will move DNS from old cluster to new cluster, thus completing the move.
 
-Move DNS from old cluster to new cluster, thus completing the move.
+## 9. Cleanup the old cluster and NFS VM, preserving anything we want from there
```

Comment:
>preserving anything we want from there

such as?? (I guess some examples could be useful here, maybe some specific logs?